### PR TITLE
Bloom expansion config

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

File: src/rebloom.c [hunk 1]
```diff
@@ -129,6 +130,15 @@ static int BFReserve_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv,
         return RedisModule_ReplyWithError(ctx, "ERR bad capacity");
     }
 
+    long long expansion = BF_DEFAULT_EXPANSION;
+    int ex_loc = RMUtil_ArgIndex("EXPANSION", argv, argc);    
+    if (ex_loc != -1) {
+        if (RedisModule_StringToLongLong(argv[ex_loc + 1], &expansion) != REDISMODULE_OK) {
+            RedisModule_ReplyWithError(ctx, "ERR bad expansion");
```
Comment (rafie) [inline src/rebloom.c]: Should be:
```
return RedisModule_ReplyWithError(ctx, "ERR bad expansion");
```
Contrary to correct intuition, methods should't return REDISMODULE_ERR when faced with an error. Thus, RedisModule_ReplyWithError returns REDISMODULE_OK.

Comment (ashtul) [reply src/rebloom.c]: @rafie 
You are correct and it is all over the file.
Talking this opportunity to fix this.

File: docs/Bloom_Commands.md [hunk 2]
```diff
@@ -34,6 +34,13 @@ filter entry.
     been exceeded. Performance will degrade linearly as the number of entries
     grow exponentially.
 
+Optional parameters:
+
+* **expansion**: When a new filter is created, its size will be the size of the
```
Comment (K-Jo) [inline docs/Bloom_Commands.md]: Can you elaborate on the when?  I believe it's not really when it's created, but "when a threshold is reached" ... "an additional filter is created which size ...".

File: docs/Bloom_Commands.md [hunk 3]
```diff
@@ -34,6 +34,13 @@ filter entry.
...
+current filter multiplied by `expansion`.
+Default expansion value is 2. New sub-filter size is (2 * expansion) of the
```
Comment (K-Jo) [inline docs/Bloom_Commands.md]: sub-filter is nowhere else defined.  related to comment above

**Pull request merged**